dbms/tests/queries/0_stateless/00397_tsv_format_synonym.sql

```diff
@@ -0,0 +1,11 @@
+SELECT arrayJoin([1, 2, 3]), 'hello', 'world' FORMAT TabSeparated;
+SELECT arrayJoin([1, 2, 3]), 'hello', 'world' FORMAT TSV;
+
+SELECT arrayJoin([1, 2, 3]), 'hello', 'world' FORMAT TabSeparatedWithNames;
+SELECT arrayJoin([1, 2, 3]), 'hello', 'world' FORMAT TSVWithNames;
+
+SELECT arrayJoin([1, 2, 3]), 'hello', 'world' FORMAT TabSeparatedWithNamesAndTypes;
+SELECT arrayJoin([1, 2, 3]), 'hello', 'world' FORMAT TSVWithNamesAndTypes;
+
+SELECT arrayJoin([1, 2, 3]), 'hello', 'world' FORMAT TabSeparatedRaw;
+SELECT arrayJoin([1, 2, 3]), 'hello', 'world' FORMAT TSVRaw;
```

Comment:
I suggest to add aliases for columns to test WithNames formats.
Because auto-generated column names are implementation specific and could change in future.